### PR TITLE
Adds MoMMI flavoured adminbus autoborger

### PR DESCRIPTION
**Pull request by ghost**
[content]
Idea by @drawsstuff, possible sprite too soon.
:cl:
 * rscadd: Malfunctioning AIs can now spawn a MoMMI flavoured autoborger, if the gods are willing.

### CODE REVIEW

**Review by Kurfursten — CHANGES_REQUESTED** [review]
Get rid of ALL this copypaste and just make a new proc that handles the transformation itself, then override that. I believe you can get this down to 7 total lines of code.

**Comment by DeityLink** [comment]
Are the mommi free-willed or auto-emagged? in the latter case, they should be slaved to the AI and you should have them added to the AI's faction like I did in https://github.com/vgstation-coders/vgstation13/pull/30161/files

**Comment by drawsstuff** [comment]
> Are the mommi free-willed or auto-emagged? in the latter case, they should be slaved to the AI and you should have them added to the AI's faction like I did in https://github.com/vgstation-coders/vgstation13/pull/30161/files

They should probably be the standard keeper lawset with the usual [ERORR ACCOMPLISH OBJECTIVES AT ALL COSTS].

**Comment by DeityLink** [comment]
so....emagged?

**Comment by Blithering** [comment]
More important question, does this call for a third eye glow colour?

**Comment by ghost** [comment]
> Get rid of ALL this copypaste and just make a new proc that handles the transformation itself, then override that. I believe you can get this down to 7 total lines of code.

Done

**Comment by drawsstuff** [comment]
> so....emagged?

No. Emagged mommis follow syndiemov or whatever you wanna call it, and have access to a stun arm.

**Comment by Eneocho** [comment]
> 
> 
> More important question, does this call for a third eye glow colour?

I'm torn on that one. On the one hand, it'd go along the concept of MoMMIs being blatantly obvious when not KEEPERs. On the other hand, that would disrupt the early stealth stage on an autoborger Malf round, since you could see the MoMMI waddling around with the purple eyes.

File: code/game/machinery/autoborger.dm [hunk 1]
```diff
@@ -214,3 +217,10 @@
 				force_borg_module = sel_mod
 	interact(usr)
 	return 1
+
+/obj/machinery/autoborger/mommi
+	name = "Autimatic Crab Factory 5000"
```
Comment (Eneocho) [inline code/game/machinery/autoborger.dm:222]:
```suggestion
	name = "Autismatic Crab Factory 5000"
```

**Review by Kurfursten — APPROVED** [review]

**Comment by Anton-Kr** [comment]
Now make it not adminbus.

**Comment by drawsstuff** [comment]
> Now make it not adminbus.

Sure, for 200 points

**Pull request merged**